android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -1241,7 +1241,7 @@
 		"cost": 425,
 		"requiredTech": "Lasers",
 		"requiredResource": "Aluminum",
-		"uniques": ["[100]% chance to intercept air attacks", "[+150]% Strength <vs [Bomber] units>", "[+150]% Strength <vs [Helicopter] units>"],
+		"uniques": ["Can perform Air Sweep", "[100]% chance to intercept air attacks", "[+150]% Strength <vs [Bomber] units>", "[+150]% Strength <vs [Helicopter] units>"],
```

Comment:
I'd rather put this unique on the unit type than the unit btw

Reply:
Done